uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/WorkPatternTest.php

```diff
@@ -502,5 +505,26 @@ public function testgetWorkDayTypeForDateShouldHaveCorrectDayTypeForPatternsWith
       new DateTime('2016-08-15'), $start
     ));
   }
-
+  public function testGetWorkDayType() {
+    //create a contact with valid contract and a Work pattern assigned
+    $contact = ContactFabricator::fabricate();
+    $periodStartDate = date('2016-01-01');
```

Comment:
You don't really the date() function here, right? You can do the same with just: `$periodStartDate = '2016-01-01';`